aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -233,35 +233,27 @@ module aptos_framework::aptos_governance {
         governance_config.required_proposer_stake = required_proposer_stake;
 
         if (std::features::module_event_migration_enabled()) {
-            event::emit(
-                UpdateConfig {
+            event::emit(UpdateConfig {
                     min_voting_threshold,
                     required_proposer_stake,
                     voting_duration_secs
-                },
-            )
+                },)
```

Comment:
I feel like the symmetry is lost here because `)` was moved up. I am pretty sure rustfmt e.g., would put this on the next line.